### ISSUE TITLE
Datasets don't appear on the project page

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5443]
**Datasets don&#39;t appear on the project page**

The following datasets have the correct project identifiers:
https://doi.org/10.15468/ukuctu
https://doi.org/10.15468/w6p22g
https://doi.org/10.15468/ek8q2b

however, they don't appear here: https://www.gbif.org/project/CESP2023-006/croment#datasets

What could be happening?

-----

Github user: @ManonGros
User: [See in registry](https://www.gbif.org/api/feedback/user/0f7d83ed2cdea43a5ad5ad7732944af7:38e963844b44d2d88715f7f6b698b9d6c935219b8cb8bbb31bd4c5c7890bf6323311a89b10dc0eafe8e05be96a5bc8e61acff756b04cd7a902a0ff3a3d34c4cfaf3131cf18ba7b430601bc904f5f44092bd75f18d38e602c4d1b1ad883a139a2f62d1422ad5fe18730d2bf61606f966048a3cd570ce4c6e827f52e7f1ad09a45d53a3709fb934917a36fa06884f6d555) - [Send email](https://www.gbif.org/api/feedback/user/mailto/0f7d83ed2cdea43a5ad5ad7732944af7:38e963844b44d2d88715f7f6b698b9d6c935219b8cb8bbb31bd4c5c7890bf6323311a89b10dc0eafe8e05be96a5bc8e61acff756b04cd7a902a0ff3a3d34c4cfaf3131cf18ba7b430601bc904f5f44092bd75f18d38e602c4d1b1ad883a139a2f62d1422ad5fe18730d2bf61606f966048a3cd570ce4c6e827f52e7f1ad09a45d53a3709fb934917a36fa06884f6d555)
System: Safari 17.6.0 / Mac OS X 10.15.7
Referer: https://www.gbif.org/project/CESP2023-006/croment
Window size: width 1731 - height 926
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-08-30T13:07:51.919Z',to:'2024-08-30T13:13:51.919Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2024-08-30T13:07:51.919Z',to:'2024-08-30T13:13:51.919Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
api call returns 1 https://api.gbif.org/v1/dataset/search?project_id=CESP2023-006

**Comment by CecSve** [comment]
@MortenHofft should this then be moved to the registry repo perhaps? I guess it is a backend issue?

**Comment by MortenHofft** [comment]
Yes it sounds to me like it is a backend issue. So I tagged it. I strive try to tag things as they come in as a service to those that have to fix it. But I rarely move issues around or tag people. So there are [many registry issues](https://github.com/gbif/portal-feedback/labels/registry)
